docs/configuration.asciidoc

```diff
@@ -617,3 +626,36 @@ The default unit for this option is `ms`
 Sets the logging level for the agent.
 
 Valid options: `Critical`, `Error`, `Warning`, `Info`, `Debug`, `Trace` and `None` (`None` disables the logging).
+
+[[config-all-options-summary]]
+=== All options summary
+
+[options="header"]
+|============
+| Option name | Is reloadable? | Keywords
+| <<config-capture-body,`CaptureBody`>> | Yes | HTTP, Performance
+| <<config-capture-body-content-types,`CaptureBodyContentTypes`>> | Yes | HTTP, Performance
+| <<config-capture-headers,`CaptureHeaders`>> | No | HTTP, Performance
+| <<config-central-config,`CentralConfig`>> | No | Core
+| <<config-environment,`Environment`>> | No | Core
+| <<config-flush-interval,`FlushInterval`>> | No | Reporter
+| <<config-global-labels,`GlobalLabels`>> | No | Core
+| <<config-log-level,`LogLevel`>> | No | Supportability
+| <<config-max-batch-event-count,`MaxBatchEventCount`>> | No | Reporter
+| <<config-max-queue-event-count,`MaxQueueEventCount`>> | No | Reporter
+| <<config-metrics-interval,`MetricsInterval`>> | No | Reporter
+| <<config-sanitize-field-names,`SanitizeFieldNames`>> | No | Core
+| <<config-secret-token,`SecretToken`>> | No | Reporter
+| <<config-server-urls,`ServerUrls`>> | No | Reporter
+| <<config-service-name,`ServiceName`>> | No | Core
+| <<config-service-version,`ServiceVersion`>> | No | Core
+| <<config-span-frames-min-duration,`SpanFramesMinDuration`>> | No | Stacktrace, Performance
+| <<config-stack-trace-limit,`StackTraceLimit`>> | No | Stacktrace, Performance
+| <<config-transaction-max-spans,`TransactionMaxSpans`>>  | Yes | Performance, Core
+| <<config-transaction-sample-rate,`TransactionSampleRate`>> | Yes | Performance, Core
+|============
+
+NOTE: Option being reloadable means that the setting can be changed after agent's start.
+For example some reloadable options can be changed
+via {apm-app-ref}/agent-configuration.html[APM Agent configuration]).
```

Comment:
Thanks for using `apm-app-ref` here!! 😊

Reply:
I just copied [your changes](https://github.com/elastic/apm-agent-dotnet/pull/574/files#diff-01c1a79a5a1e1b67003a1885365f2cf5R251) :)